src/test/java/org/takes/facets/auth/PsBasicTest.java

```diff
@@ -185,11 +189,59 @@ public void handleMultipleHeadersWithInvalidContent() throws Exception {
     }
 
     /**
+     * Test {@link org.takes.facets.auth.PsBasic.Default#enter} retunrs
+     * a valid user.
+     * With user and pass ok.
+     */
+    @Test
+    public void returnsCorrectUrn() {
+        final String user = "Admin";
+        final String pass = "123";
+        final String urn = "urn:basic:admin";
+        final List<User> usrs =
+            new ArrayList<User>(1);
+        usrs.add(new User(user, pass, urn));
+        final Opt<Identity> identity = new PsBasic.Default(usrs)
+            .enter(user, pass);
+        MatcherAssert.assertThat(identity.has(), Matchers.is(true));
```

Comment:
@lautarobock this line is redundant, the next assertion will fail if there is no URN. also `identity` is a one-time var, see  http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#one-time-variables